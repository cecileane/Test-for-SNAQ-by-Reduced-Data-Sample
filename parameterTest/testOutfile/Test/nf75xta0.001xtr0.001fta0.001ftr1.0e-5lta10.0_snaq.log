optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.001,
                       xtolAbs=0.001, xtolRel=0.001.
 max number of failed proposals = 75, liktolAbs = 10.0.
rootname for files: nf75xta0.001xtr0.001fta0.001ftr1.0e-5lta10.0_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 30241
seed: 30241 for run 1
Sat Oct  1 17:27:39 2016

 BEGIN SNaQ for run 1, seed 30241 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 89 (failures: 75) with -loglik=128.22643 and ht_min=[0.30771,8.87902,0.4599,0.66107,1.73899]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 26		 26		 0	 |	 0		 0		 --
mvtarget	 28		 28		 1	 |	 0		 0		 --
chdir		 5		 5		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 29		 24		 1	 |	 1		 1		 --
Total		 89		 84		 3	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 128.2264279768451
(3,(((1,#H7:::0.3077087543589899):8.879019849836414,2):0.6610725671447303,4):0.45989808428584517,(5,(6)#H7:::0.6922912456410101):1.7389873904932542);
---------------------
seed: 52594 for run 2
Sat Oct  1 17:27:45 2016

 BEGIN SNaQ for run 2, seed 52594 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 123 (failures: 75) with -loglik=158.61503 and ht_min=[0.31865,0.00549,1.11438,0.32447,10.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 5		 5		 2	 |	 NA 		 NA 		 NA 
mvorigin	 27		 27		 1	 |	 5		 5		 --
mvtarget	 37		 37		 0	 |	 5		 5		 --
chdir		 10		 10		 0	 |	 10		 10		 --
delete		 0		 0		 0	 |	 1		 1		 --
nni		 44		 35		 1	 |	 0		 0		 --
Total		 123		 114		 4	 |	 21		 21		 2
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 0.1

 FINISHED SNaQ for run 2, -loglik of best 158.6150256863684
(3,((1,(2,#H7:::0.31864503575383474):10.0):1.1143839222012684,(5,6):0.3244651543229791):0.00548745321142531,(4)#H7:::0.6813549642461653);
---------------------

Sat Oct  1 17:27:52 2016
MaxNet is (3,(((1,#H7:::0.3077087543589899):8.879019849836414,2):0.6610725671447303,4):0.45989808428584517,(5,(6)#H7:::0.6922912456410101):1.7389873904932542); 
with -loglik 128.2264279768451
