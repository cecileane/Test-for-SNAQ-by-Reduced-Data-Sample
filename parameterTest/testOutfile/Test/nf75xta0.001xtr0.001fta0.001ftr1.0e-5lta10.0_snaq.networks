(3,(((1,#H7:::0.3077087543589899):8.879019849836414,2):0.6610725671447303,4):0.45989808428584517,(5,(6)#H7:::0.6922912456410101):1.7389873904932542);, with -loglik 128.2264279768451 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,(((1)#H7:::0.7150067434291001,2):1.514822943102537,4):0.48792138669199087,(5,(6,#H7:::0.28499325657089997):10.0):0.5222727054117268);, with -loglik 126.74438813153336
(3,(4)#H7:::0.6154929592733516,(5,(6,(1,(#H7:::0.3845070407266484,2):7.898880909301712):1.1797153444852608):0.21658050819647087):0.024780511846986434);, with -loglik 171.91941335071573
(3,(#H7:::0.4302104722882283,4):1.8451052559523828,(5,(6,(1,(2)#H7:::0.5697895277117717):4.400650756376644):0.15223028083117143):0.03913777873232293);, with -loglik 192.0872861678527
((3)#H7:::0.5404786364490572,((1,(6,(5,#H7:::0.45952136355094275):1.355705617073628):1.0073185183896043):0.10423738683797941,2):0.11138055849161906,4);, with -loglik 214.69290639250846
(3,(((1,(6,#H7:::0.4874026198922201):5.226181429235515):0.04929336927792822,2):0.20937105166811237,4):1.087664673496748,(5)#H7:::0.5125973801077799);, with -loglik 231.35116955209168
