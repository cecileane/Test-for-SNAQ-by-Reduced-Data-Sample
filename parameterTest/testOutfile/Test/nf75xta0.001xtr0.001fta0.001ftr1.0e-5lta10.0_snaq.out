(3,(((1,#H7:::0.3077087543589899):8.879019849836414,2):0.6610725671447303,4):0.45989808428584517,(5,(6)#H7:::0.6922912456410101):1.7389873904932542); -Ploglik = 128.2264279768451
 Dendroscope: (3,(((1,#H7):8.879019849836414,2):0.6610725671447303,4):0.45989808428584517,(5,(6)#H7):1.7389873904932542);
 Elapsed time: 12.962426381 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (3,(((1,#H7:::0.3077087543589899):8.879019849836414,2):0.6610725671447303,4):0.45989808428584517,(5,(6)#H7:::0.6922912456410101):1.7389873904932542);, with -loglik 128.2264279768451
 (3,((1,(2,#H7:::0.31864503575383474):10.0):1.1143839222012684,(5,6):0.3244651543229791):0.00548745321142531,(4)#H7:::0.6813549642461653);, with -loglik 158.6150256863684
-------
